Publisher/Tests/Mocks/MockAblyPublisherService.swift

```diff
@@ -44,9 +44,9 @@ class MockAblyPublisherService: AblyPublisherService {
         sendEnhancedAssetLocationUpdateParamTrackable = trackable
         sendEnhancedAssetLocationUpdateParamCompletion = completion
     }
-
-    var stopCalled: Bool = false
-    func stop() {
-        stopCalled = true
+    
+    var closeCalled: Bool = false
+    func close(completion: @escaping ResultHandler<Void>) {
```

Comment:
What's the `completion` argument for?

Reply:
You are right. The `close` method was not covered by the tests.
Updated:
https://github.com/ably/ably-asset-tracking-cocoa/blob/feature/ATS-266-Publisher_stop/Publisher/Tests/DefaultPublisher/DefaultPublisherTests.swift#L525